wdl-engine/src/stdlib/quote.rs

```diff
@@ -79,17 +77,21 @@ mod test {
         let elements: Vec<_> = value
             .as_array()
             .unwrap()
-            .elements()
+            .as_slice()
             .iter()
             .map(|v| v.as_string().unwrap().as_str())
             .collect();
-        assert_eq!(elements, [r#""1.0""#, r#""1.1""#, r#""1.2""#]);
+        assert_eq!(elements, [
+            r#""1.000000""#,
```

Comment:
Is this desirable?

Reply:
The precision? The spec requires six decimal places for displaying floats.

Reply:
That feels bad. Perhaps there is a good reason for it?

Reply:
It's mentioned in the section covering how floats are [coerced to string in a placeholder](https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#expression-placeholder-coercion).

Unfortunately no justification given.

Reply:
Okay, I'm satisfied at any rate.